packages/babel-parser/test/fixtures/jsx/html-entities/html5-entities/input.js

```diff
@@ -0,0 +1,3 @@
+<div>
+  &apos; &lpar; &middot; &#xB7; &amp; &reg; &centerdot; &CenterDot; &#x1D4B2; &rbrace;
```

Comment:
Could you also add `&CounterClockwiseContourIntegral;` to this test suite?

One of the longest named character references in HTML4 is `&thetasym;`, which has a length of 10 characters. This test case ensures that Babel parser correctly supports the longest named character references defined in HTML5.